scripts/Electronics.zs

```diff
@@ -0,0 +1,473 @@
+
+// Qubit CPU Wafer * 1
+<recipemap:chemical_reactor>.findRecipe(1920, [<metaitem:wafer.nano_central_processing_unit>, <metaitem:quantumeye> * 2], [<liquid:gallium_arsenide> * 288]).remove();
+
+// NOR Wafer * 8
+<recipemap:laser_engraver>.findRecipe(7680, [<metaitem:wafer.neutronium>, <metaitem:glass_lens.pink>], null).remove();
+
+// NOR Wafer * 4
+<recipemap:laser_engraver>.findRecipe(1920, [<metaitem:wafer.naquadah>, <metaitem:glass_lens.pink>], null).remove();
+
+// NOR Wafer * 1
+<recipemap:laser_engraver>.findRecipe(480, [<metaitem:wafer.glowstone>, <metaitem:glass_lens.pink>], null).remove();
+
+// Qubit CPU Wafer * 1
+<recipemap:large_chemical_reactor>.findRecipe(1920, [<metaitem:wafer.nano_central_processing_unit>, <metaitem:quantumeye> * 2], [<liquid:gallium_arsenide> * 288]).remove();
+
+// Qubit CPU Wafer * 1
+<recipemap:chemical_reactor>.findRecipe(1920, [<metaitem:wafer.nano_central_processing_unit>, <metaitem:dustIndiumGalliumPhosphide>], [<liquid:radon> * 50]).remove();
+
+// Qubit CPU Wafer * 1
+<recipemap:large_chemical_reactor>.findRecipe(1920, [<metaitem:wafer.nano_central_processing_unit>, <metaitem:dustIndiumGalliumPhosphide>], [<liquid:radon> * 50]).remove();
+
+
+//-------------ADD NEW RECIPES-------------------
+
+//-------------SILICON PURIFICATION--------------
+mixer.recipeBuilder()
+.inputs([<ore:dustSilicon>])
+.fluidInputs([
+    <liquid:hydrofluoric_acid> * 1000,
+    <liquid:ethanol> * 100
+    ])
+.fluidOutputs([<liquid:silicon_solution> * 1000])
+.duration(100)
+.EUt(20)
+.buildAndRegister();
+
+chemical_reactor.recipeBuilder()
+.notConsumable([<metaitem:emitter.mv>])
+.fluidInputs([
+    <liquid:silicon_solution> * 1000
+    ])
+.outputs([<metaitem:dustNanosilicon> * 1])
+.duration(250)
+.EUt(120)
+.buildAndRegister();
+
+sifter.recipeBuilder()
+.inputs([<metaitem:dustNanosilicon> * 1])
+.outputs([<metaitem:dustRefinedsilicon> * 1])
+.duration(400)
+.EUt(32)
+.buildAndRegister();
+
+//--------------SILICON BOULE------------------
+ebf.recipeBuilder()
+.property("temperature", 1784)
+.inputs([
+    <gregtech:meta_seed_crystal:99> * 1,
+    <metaitem:dustRefinedsilicon> * 32
+    ])
+.outputs([<metaitem:boule.silicon>])
+.duration(9000)
+.EUt(120)
+.buildAndRegister();
+
+//--------------LITHOGRAPHY PLATE---------------
+assembler.recipeBuilder()
+.fluidInputs([<liquid:dye_black> * 100])
+.inputs([<ore:plateGlass>])
+.outputs([<contenttweaker:lithography_mask>])
+.duration(600)
+.EUt(24)
+.buildAndRegister();
+
+
+//--------------SILICON WAFERS-----------------
+cutting_saw.recipeBuilder()
+.fluidInputs([<liquid:lubricant> * 20])
+.inputs([<metaitem:boule.silicon>])
+.outputs([
+    <metaitem:wafer.silicon> * 16,
+    <gregtech:meta_seed_crystal:99>
+    ])
+.duration(400)
+.EUt(64)
+.buildAndRegister();
+
+cutting_saw.recipeBuilder()
+.fluidInputs([<liquid:distilled_water> * 60])
+.inputs([<metaitem:boule.silicon>])
+.outputs([
+    <metaitem:wafer.silicon> * 16,
+    <gregtech:meta_seed_crystal:99>
```

Comment:
Do these have an ore dictionary name?
If they do you should use the ore dict entry instead
If they don't I really should add them in the mod